mayastor/src/pool.rs

```diff
@@ -131,3 +136,32 @@ impl From<Pool> for rpc::Pool {
         }
     }
 }
+
+/// PoolArgs is used to translate the input for the grpc
+/// Create/Import requests which comtains name, uuid & disks.
+/// This help us avoid importing grpc structs in the actual lvs mod
+#[derive(Clone, Debug)]
+pub struct PoolArgs {
+    pub name: String,
+    pub disks: Vec<String>,
```

Comment:
can we add getters rather than having these as pub?

Reply:
This will require some cleanup in `subsys/config` code will take that in separate PR.